### PR TITLE
Added fitInsideVertically support for bar chart and line chart tooltips.

### PR DESCRIPTION
**Pull request by thebuggycoder**
This PR provides a fix for #225
Also fixes a minor renaming bug that was in PR #236 

Changes added -

Added new parameter fitInsideVertically to BarTouchTooltipData and LineTouchTooltipData for fitting the tooltip vertically inside the charts.
Renamed fitInsideTheChart to fitInsideHorizontally to avoid confusion.

### CODE REVIEW

File: lib/src/chart/bar_chart/bar_chart_data.dart [hunk 1]
```diff
@@ -235,7 +235,7 @@ class BarChartRodData {
     BorderRadius borderRadius,
     this.backDrawRodData = const BackgroundBarChartRodData(),
     this.rodStackItem = const [],
-  }) :borderRadius = normalizeBorderRadius(borderRadius, width);
+  }) : borderRadius = normalizeBorderRadius(borderRadius, width);
```
Comment (imaNNeoFighT) [inline lib/src/chart/bar_chart/bar_chart_data.dart]: Please prevent to left this kind of changes,
they just make it hard to review.
If you want to reformat somewhere, make a separate PR.
We shouldn't do reformat inside our feature request.

**Comment by imaNNeoFighT** [comment]
And could you please update the [LineChart](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/line_chart.md) and the [BarChart](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/line_chart.md) documentations?

File: lib/src/chart/bar_chart/bar_chart_data.dart (hunk 1 above)
Comment (thebuggycoder) [reply lib/src/chart/bar_chart/bar_chart_data.dart]: My apologies. Didn't realize this got changed. 
It must have been automatically formatted by VS Code. I'll remove the space if you want.

**Comment by thebuggycoder** [comment]
Sure thing. Will update ASAP.

**Comment by thebuggycoder** [comment]
Documentation is updated now. I didn't change the space formatting issue as it was automatically added by VS Code each time I reverted it and saved the file. Please let me know if you want it removed and I'll update it manually.

Will make sure such changes don't happen in the future. Thanks! 😄

**Comment by imaNNeoFighT** [comment]
Thank you,
You can double-check it before each commit.
I will merge it.
Cheers!

**Comment by imaNNeoFighT** [comment]
Oh man
you didn't set the target branch to `dev` :))))

**Comment by imaNNeoFighT** [comment]
I've reverted it.

**Comment by imaNNeoFighT** [comment]
Please make another PR and target `dev`.
Thanks!

**Comment by thebuggycoder** [comment]
Done! Please refer to #240 for new PR to dev branch. Apologies for all the confusion. 😄
Cheers!

**Comment by imaNNeoFighT** [comment]
No problem man,
Thanks for contributing,
Cheers!
